### ISSUE TITLE
Add option to configure grpc.max_receive_message_length

### ISSUE THREAD

**Issue by doochik** [yandex-cloud/ydb-nodejs-sdk#6]
I have query result more than 4MB and get the error 
`Error: 8 RESOURCE_EXHAUSTED: Received message larger than max (4324396 vs. 4194304)`

I think you should add an option to configure `grpc.max_receive_message_length`.

https://github.com/yandex-cloud/ydb-nodejs-sdk/blob/8fb772f1518a209455aa5bf219fb5150bdae8716/src/utils.ts#L88

Example
```
new grpc.Client(host, grpc.credentials.createInsecure(), {
    "grpc.max_receive_message_length": 1024 * 1024 * 100,
    "grpc.max_send_message_length": 1024 * 1024 * 100
});
```

**Comment by tsufiev** [comment]
@doochik Please, take a look at https://github.com/yandex-cloud/ydb-nodejs-sdk/pull/75

It should make possible to create driver like
```
    const options = {
        "grpc.max_receive_message_length": 1024 * 1024 * 100,
        "grpc.max_send_message_length": 1024 * 1024 * 100
    }
    const driver = new Driver(entryPoint, dbName, authService, {clientOptions: options});
```
and then enjoy increased limits for SchemaClient and TableClient calls.

**Comment by doochik** [comment]
looks good!

**Comment by tsufiev** [comment]
Published as ydb-sdk@2.1.0
